### ISSUE TITLE
Update example to the new configurator

### ISSUE THREAD

**Issue by Chris00** [ocaml/dune#1009]
The example [Defining a library with C stubs using pkg-config](https://dune.readthedocs.io/en/latest/quick-start.html#defining-a-library-with-c-stubs-using-pkg-config) in the documentation should be adapted to the new Configurator.  In particular, it should not be using `Stdio` nor `Base`.

**Comment by prometheansacrifice** [comment]
@rgrinberg The following worked for me.

```
open Configurator.V1.Flags
module C = Configurator.V1

let () =
  C.main ~name:"foo" (fun c ->
    let default : C.Pkg_config.package_conf =
      { libs   = ["-lgst-editing-services-1.0"]
      ; cflags = []
      }
    in
    let conf =
      match C.Pkg_config.get c with
      | None -> default
      | Some pc ->
         match (C.Pkg_config.query pc ~package:"gst-editing-services-1.0") with
         | None -> default
         | Some deps -> deps
    in

    let write_sexp fn eles =
      write_sexp fn eles
    in
    
    write_sexp "c_flags.sexp"         conf.cflags;
    write_sexp "c_library_flags.sexp" conf.libs)
```

Good enough for the docs?

**Comment by rgrinberg** [comment]
Looks good. I'd get rid of the `open` and the:

```
   let write_sexp fn eles =
      write_sexp fn eles
    in
```

Opens make things hard to read in docs and I'm not sure what's the point of that snippet above.

**Comment by prometheansacrifice** [comment]
Yeah, that's redundant. Raising a PR soon.

**Comment by emillon** [comment]
This is closed by #1224. Thanks!
